CHANGELOG.md

```diff
@@ -13,6 +27,7 @@
 - Changed implementation of `DeviceInfoProvider` provider in order to make device's id persistent
 - Changed implementation of `DeviceChecker` to increase performance on devices with A12 Bionic SoC
 - Removed `MapboxNavigation` dependency
+- Updated source code of `Reachability.swift` dependency
```

Comment:
@dersim-davaod is this correct that this was changed in 0.6.0 instead of 0.5.0?

Reply:
Yes, changes were merged after 0.5.0 release.

Reply:
@chezzdev @dersim-davaod  fix for changelog https://github.com/mapbox/mapbox-vision-ios/pull/189